gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/ApiDuplicatorServiceImpl.java

```diff
@@ -107,12 +107,16 @@ public ApiEntity createWithImportedDefinition(String apiDefinitionOrURL, String
         String apiDefinition = fetchApiDefinitionContentFromURL(apiDefinitionOrURL);
         try {
             final JsonNode jsonNode = objectMapper.readTree(apiDefinition);
+
+            String apiId = UuidString.generateRandom();
+            ((ObjectNode) jsonNode).put("id", apiId);
```

Comment:
I wonder if that break the behavior of the line 3 of our "Import rules" GoogleSheets file : `200 : API is created with provided ID`

Reply:
That was messing with the test suite indeed. https://github.com/gravitee-io/gravitee-api-management/pull/1005/commits/bd61499eda4c7492d00fd3a04699225b476448ca makes tests pass.

Reply:
ok